lib/sepa_king/transaction.rb

```diff
@@ -4,6 +4,8 @@ class Transaction
     include ActiveModel::Validations
     extend Converter
 
+    DEFAULT_REQUESTED_DATE = Date.new(1999, 1, 1).freeze
```

Comment:
Hm, with _next possible date_ I didn't mean _tomorrow_. For CreditTransfer it should be the next TARGET day, for DirectDebit it depends on RCUR/FRST/COR1.

But no problem, the name of the constant is not as important for me - leave it as it is :)